docs/developer/environment-setup.md

```diff
@@ -0,0 +1,198 @@
+# Getting started
```

Comment:
For all the documentation files that have been copied over from the Althea branch, please cherry-pick them so that I retain authorship.

Pulling them downstream is fine, editing the names and links to match Somm's codebase great. Please fix the authorship of the initial commit.

Reply:
done